contrib/update/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.7.2-alpine
+
+WORKDIR /serenata-update
+
+COPY requirements.txt requirements.txt
+
+RUN apk update && \
+    apk add openssh-keygen openssl-dev && \
+    /usr/bin/ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa && \
```

Comment:
Oh, now I see why you hookup those files inside of the droplet.
This way you have a sort of one time SSH key running things in your droplets. Although it works, I wouldn't recommend that. You lose track of who's running what in your cloud account.

Reply:
True story. But that's the best I could think at the moment. Anyway, if we can sort out the SSH Key issue with another strategy (downloading from GitHub or using `ssh_key_ids`) we tackle that too ; )

Reply:
Also fixed by fb58719